hackclub.com.yaml

```diff
@@ -1390,6 +1390,10 @@ clutter:
   ttl: 300
   type: CNAME
   value: a.selfhosted.hackclub.com.
+clubs: # hya@yip.cat U08E3UAUHFS
+  - ttl: 600
+    type: CNAME
+    value: clubs.
```

Comment:
you need to change the value to where you are hosting the site. where is your site being hosted?